validator/validator.py

```diff
@@ -32,7 +32,7 @@ def validate(self):
             errors_on_key = {}
 
             for rule in rules_array:
-                if not rule(req_value):
+                if not rule.check(req_value):
```

Comment:
can you revert this the way it was?

and encapsulate check into __call__ method (for future usages)